src/controllers/propertyEndpoints/propertyendpoint.js

```diff
@@ -0,0 +1,93 @@
+import Model from '../../models/model';
+
+const propertyModel = new Model('properties');
+export const createProperty = async (req, res) => {
+  const { id } = req.user.user;
+  const {
+    image,
+    title,
+    price,
+    status,
+    address,
+    city,
+    neighbourhood,
+    LGA,
+    ZIP,
+    numberOfBaths,
+    numberOfBeds,
+    landSize,
+  } = req.body;
+  const columns = 'agent_id, image_url, title, price, status, address, city, neighbourhood, lga, zip_code, number_of_baths, number_of_beds, land_size';
+  const values = `'${id}', '${image}', '${title}', '${price}', '${status}', '${address}', '${city}', '${neighbourhood}', '${LGA}', '${ZIP}', '${numberOfBaths}', '${numberOfBeds}', '${landSize}'`;
+  try {
+    const data = await propertyModel.insertWithReturn(columns, values);
+    res.status(201).json(data.rows);
+  } catch (err) {
+    res.status(500).json({ messages: err.stack.messages });
+  }
+};
+export const allProperties = async (req, res) => {
```

Comment:
Your controllers should be named according to the action they perform, this is better named `getAllProperties`